homeassistant/helpers/entity_registry.py

```diff
@@ -268,6 +268,18 @@ def as_partial_dict(self) -> dict[str, Any]:
             "unique_id": self.unique_id,
         }
 
+    @cached_property
+    def extended_dict(self) -> dict[str, Any]:
+        """Return a extended dict representation of the entry."""
+        return {
+            **self.as_partial_dict,
```

Comment:
These should almost always be in the cache since it will get populated on first connect